cmd/node/config/api.toml

```diff
@@ -93,3 +93,8 @@
          # /transaction/:txhash will return the transaction in JSON format based on its hash
          { Name = "/:txhash", Open = true },
 	]
+
+[APIPackages.history]
```

Comment:
This should be handled by `/transaction/:txhash` route, the client of this API shouldn't care about the type of the node that's running

Reply:
done.